Makefile

```diff
@@ -0,0 +1,23 @@
+RPM_BASE:=$(shell rpmspec --query bfscripts.spec)
+SRPM_NAME:=$(subst .noarch,,$(RPM_BASE)).src.rpm
+GIT_FILES:=$(shell git ls-files -co --exclude-standard)
+
+.PHONY: all clean
+all: $(SRPM_NAME)
+
+clean:
+	rm -r RPMBUILD
+	rm -r git_dir_pack
+	rm -f mlxbf-bfscripts*.src.rpm
+
+RPMBUILD/SOURCES/mlxbf-bfscripts.tar.gz: $(GIT_FILES)
+	mkdir -p RPMBUILD/SOURCES
```

Comment:
Just FYI, command "git archive --format=tar --prefix=xxx/ master | gzip > xxx.tar.gz" might also do the work. up to you to pick the one you like

Reply:
Note that this replicates the git_dir_pack behavior that rpkg uses.